disnake/guild.py

```diff
@@ -2496,12 +2497,10 @@ async def create_sticker(
         :class:`GuildSticker`
             The created sticker.
         """
-        payload: Any = {
-            "name": name,
-        }
+        if description is None:
+            description = ""
 
-        if description:
-            payload["description"] = description
+        payload: CreateStickerPayload = {"name": name, "description": description}  # type: ignore
```

Comment:
This could be moved down to avoid the `# type: ignore`

Reply:
changed in 202f08df2fdbfd77baa46aac8086d200dbcc4825, thanks!